[Docs] Complete VxLAN commands documentation

link19430 opened the pull request:
#### Why I did it  
The VxLAN command documentation was incomplete, missing descriptions for several `show` and `config` commands, which could affect user understanding and operation.  

#### How I did it  
Added detailed descriptions for `show vxlan` and 'config vxlan' commands.

#### How to verify it  
Review the documentation to ensure all VxLAN-related commands are properly described and clearly explained.

link19430 committed:
[Docs] Complete VxLAN commands documentation

The VxLAN command documentation was incomplete, missing descriptions for several `show` and `config` commands, which could affect user understanding and operation.

Added detailed descriptions for `show vxlan` and 'config vxlan' commands.

mssonicbld commented:
/azp run

maipbui commented on doc/Command-Reference.md:
````diff
@@ -11997,13 +11997,254 @@ This command displays <vlan_name> configuration.
+  Total count : 1
+  ```
+
+**show vxlan remote_mac <remoteip/all>**
````
>remote_mac [](http://example.com/codeflow?start=13&length=10)

example command shows `remotemac`, seems inconsistent

link19430 replied:
Correct parameter name to be 'remotemac'

maipbui commented on doc/Command-Reference.md:
````diff
@@ -11997,13 +11997,254 @@ This command displays <vlan_name> configuration.
+- Usage:
+
+  ```
+  show vxlan remote_mac <remoteip/all>
````
>remote_mac [](http://example.com/codeflow?start=13&length=10)

same here

link19430 replied:
Correct parameter name to be 'remotemac'

link19430 committed:
[Docs] Correct VxLAN command parameter naming

The VxLAN command documentation contained incorrect parameter names:
- `show vxlan remote_vni` should be `show vxlan remotevni`
- `show vxlan remote_mac` should be `show vxlan remotemac`

These inconsistencies could cause confusion for users following the documentation.
- Updated the command names in the documentation to match the correct CLI syntax.

mssonicbld commented:
/azp run

mssonicbld commented:
/azp run

link19430 commented:
@maipbui Please help review, thanks

link19430 reviewed (commented):
Done

link19430 commented:
/azpw run

mssonicbld commented:
/AzurePipelines run

link19430 commented:
Hi @maipbui
Is the 'Semgrep' need to be approved to verify ?
Thanks.

link19430 commented:
Hi @maipbui 
Could you help merge it ? thanks.
